File: c/01-sequencias/main011.c
/*Desenvolva uma lógica que leia os valores de A, B e C de uma equação do
segundo grau e mostre o valor de Delta.

Data 27/09/2025 -> 6ª Dia
*/

#include <stdio.h>

int main() {

    int A;
    int B;
    int C;
    int delta;

    printf("Valor de A: ");
    scanf("%d",&A);

    printf("Valor de B: ");
    scanf("%d",&B);

    printf("Valor de C: ");
    scanf("%d",&C);

    delta = (B*B) - (4 * A * C);
    
    printf("Valor do delta e: %d",delta);
    return 0;

}

File: python/01-sequencias/main011.py
'''Desenvolva uma lógica que leia os valores de A, B e C de uma equação do
segundo grau e mostre o valor de Delta.

Data 27/09/2025 -> 6ª Dia
'''

A = int(input('Valor de A: '))
B = int(input('Valor de B: '))
C = int(input('Valor de C: '))

delta = (B*B) - (4 * A * C)

print(f'Valor do delta é: {delta}')